### PR TITLE
N14: OpenGL preprocessor

### PR DESCRIPTION
**Pull request by krupitskas**
_Part of #278 ("Newsletter 14")_

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -64,6 +64,29 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+inherit code from other templates.
+
+![An OpenGL preprocessor for Rust](opengl_preprocessor.png)
+
+Writing plain GLSL code is uncomfortable, code is quite often is duplicated, libraries
+aren't something natural for GLSL (means you can't out of the box do #include "library.glsl").
+Last point especially problematic if some constants actually originate in your
+game logic (like the number of player types). Updating these values manually in
+your shader code is repetitive and prone to both error and simple forgetfulness.
+It’s really helpful to build some kind of preprocessor for your GLSL code,
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:80]: Please replace this smart quote with a normal one:
```suggestion
It's really helpful to build some kind of preprocessor for your GLSL code,
```

**Review by 17cupsofcoffee — COMMENTED** [review]
The content looks good! There's a few grammar tweaks that would be nice to make:

"With the power of cargo build scripts full power of Tera, you can create advanced..." could be "With the full power of Cargo build scripts and Tera, you can create an advanced...".

"Last point especially problematic if..." could be "The last point is especially problematic if...".

"and create compile-time preprocessing tool..." could be "and create a compile-time preprocessing tool".

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by 17cupsofcoffee** [comment]
Content looks good, just needs the lines wrapping at 80 characters to pass the CI check.

**Comment by ozkriff** [comment]
Merged, thanks!

**Pull request merged**